cmd/root.go

```diff
@@ -9,7 +9,6 @@ import (
 
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	"k8s.io/client-go/kubernetes/scheme"
-	"k8s.io/kubectl/pkg/util/templates"
```

Comment:
With this removed, can you run `go mod tidy` to see if anything gets removed?

Reply:
Good idea but sadly nothing is removed in those files as module `k8s.io/kubectl` is used in bunch of other files.